ecosystem/indexer-grpc/indexer-grpc-file-store-backfiller/src/metrics.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{register_int_counter, IntCounter};
+use once_cell::sync::Lazy;
+
+/// Number of transactions that have been stored.
+pub static PROCESSED_VERSIONS_COUNT: Lazy<IntCounter> = Lazy::new(|| {
+    register_int_counter!(
+        "indexer_grpc_file_store_processed_versions",
```

Comment:
Let's prefix these metrics with _backfill so we can separate it from the production filestore worker metrics

Reply:
Also I don't see these metrics used anywhere

Reply:
yep... removed now; use logs to track status now.